frontend/app/src/store/__test__/main.spec.ts

```diff
@@ -934,47 +934,47 @@ describe('Main store', () => {
     expect(store.exportMetricsBountiesCsv).toBeCalledWith(body);
   });
 
-  it("I should be able to test that the signed-in user details are persisted in the local storage", async () => {
+  it('I should be able to test that the signed-in user details are persisted in the local storage', async () => {
     uiStore.setShowSignIn(true);
 
     localStorageMock.setItem('ui', JSON.stringify(uiStore));
 
     expect(uiStore.showSignIn).toBeTruthy();
     expect(localStorageMock.getItem('ui')).toEqual(JSON.stringify(uiStore));
-  })
+  });
 
-  it("I should be able to test that when signed out the user data is deleted", async () => {
-    // Shows first if signed in 
+  it('I should be able to test that when signed out the user data is deleted', async () => {

```

Comment:
@ManyRios When a user is deleted the ui data should be null or undefined,  because the ui data gets deleted from the local storage